### PR TITLE
docs(RHTAPWATCH-328): update alerting rules process

### PR DESCRIPTION
**Pull request by yftacherzog**
This change documents the process of updating app-interface with new references for the RHTAP alerting rules in case the rules are updated in the o11y repo.

### CODE REVIEW

**Comment by yftacherzog** [comment]
@Kousalya1998 @amisstea please take a look. I'm going to wait until we have the app-interface MR merged before marking this one as ready.

File: rhobs/README.md
Comment (ernesgonzalez33) [inline rhobs/README.md:9]: I think you added that "we" as a mistake in this line

Comment (yftacherzog) [reply rhobs/README.md]: done

**Review by Kousalya1998 — APPROVED** [review]
I think you have covered all the necessary points. LGTM.

File: rhobs/README.md
Comment (srivickynesh) [inline rhobs/README.md:3]:
```suggestion
[alerting](alerting) 
```
directory not found

Comment (srivickynesh) [inline rhobs/README.md:3]:
```suggestion
The rules are defined inside the [alerting](alerting) directory and are referenced using
```
this link is not there.

Comment (yftacherzog) [reply rhobs/README.md:3]: I lost track of this PR. I think I accidentally marked the ticket as done.
Yes. It was written before the directory was moved, and I think it was eventually moved elsewhere. I'll look into that.

Comment (yftacherzog) [reply rhobs/README.md:3]: ok. I see now that I replaced it with another PR instead: https://github.com/redhat-appstudio/o11y/pull/54
I'll close this one.
Thank you for reminding me :)

**Pull request closed without merging**
